packages/connect/src/api/blockchainGetCurrentFiatRates.ts

```diff
@@ -46,9 +48,13 @@ export default class BlockchainGetCurrentFiatRates extends AbstractMethod<
 
     async run() {
         const backend = await initBlockchain(this.params.coinInfo, this.postMessage);
-        return backend.getCurrentFiatRates({
-            currencies: this.params.currencies,
-            token: this.params.token,
-        });
+        return scheduleAction(
+            _signal =>
```

Comment:
`signal` param is not required, makes no sense to have it here if not used. I'll try to rework it a bit in next commit, because I fell like this timeout doesn't fit to connect and should be handled in fiat-services instead. And the constant (15000) should be defined as constant somewhere.